package.json

```diff
@@ -83,7 +83,7 @@
 	"scripts": {
 		"start": "react-scripts start",
 		"build": "npm run generate-meta-tag && react-scripts build",
-		"test": "react-scripts test",
+		"test:watch": "react-scripts test",
```

Comment:
Do we need to change this script name?

Reply:
@ericlowturing not really, I changed it so I could watch the test a bit cleaner. I'll remove it now.  
Thank you!